4b/Grains/src/Coordinator.cs

```diff
@@ -56,7 +70,8 @@ public async Task<string> RunTests()
             };
 
             await Task.WhenAll(tests);
-            testRun.TestDefinitions.UnitTests = tests.Select(t => t.Result).ToList();
+            var testDetailsResult = tests.Select(t => t.Result);
+            testRun.TestDefinitions.UnitTests = testDetailsResult.Select(tdr => tdr.UnitTestDefinition).ToList();
```

Comment:
```suggestion
            testRun.TestDefinitions.UnitTests = tests.Select(tdr => tdr.Result.UnitTestDefinition).ToList();
```

Then we can remove line 73.